src/platform/forms/tests/components/common/grid/Column.unit.spec.jsx

```diff
@@ -0,0 +1,42 @@
+// libs
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+
+import Column from 'platform/forms/components/common/grid/Column';
+
+describe('Common Column component', () => {
+  it('should render', () => {
+    const tree = render(<Column testId="12345" />);
+
+    expect(tree.getByTestId('12345')).to.not.be.null;
+    tree.unmount();
+  });
+
+  it('should render child components', () => {
+    const childComponent = <div data-testid="12345">i am a component</div>;
+
+    const tree = render(<Column>{childComponent}</Column>);
+
+    expect(tree.getByTestId('12345')).to.not.be.null;
+    tree.unmount();
+  });
+
+  it('should add classNames to output', () => {
+    const classNames = 'i-am-classname';
+
+    const tree = render(<Column classNames={classNames} testId="12345" />);
+
+    expect(tree.getByTestId('12345')).to.have.class(classNames);

```

Comment:
This could be:

```js
    const { container } = render(<Column classNames={classNames} />);

    expect(container.firstChild).to.have.class(classNames);
```

Reply:
True. Do we want the data- pattern for selecting nodes or the pattern you suggested? My preference is the data-, but happy to go either way with the right reasoning.